examples/plain-html/index.html

```diff
@@ -176,6 +183,42 @@
         <leo-button kind="filled">Primary</leo-button>
       </div>
     </leo-dialog>
+
+    <leo-navigation>
+      <leo-navigationheader slot="header">
+        <leo-icon slot="logo" name="social-brave-release-favicon-fullheight-color"></leo-icon>
+        <h1 style="font: var(--leo-font-primary-heading-h4)"><span class="logo-mark">Brave</span> Accounts</h1>
```

Comment:
```suggestion
        <h1 style="font: var(--leo-font-heading-h4)"><span class="logo-mark">Brave</span> Accounts</h1>
```

that token has been removed!